widgets/events/src/EventsTable.tsx

```diff
@@ -140,12 +140,22 @@ export default class EventsTable extends React.Component {
                     />
                     <DataTable.Column
                         label="Deployment"
+                        name="deployment_display_name"
+                        show={
+                            _.size(data.deploymentName) !== 1 &&
+                            _.size(data.nodeInstanceId) !== 1 &&
+                            _.size(data.executionId) !== 1 &&
+                            fieldsToShow.indexOf('Deployment') >= 0
+                        }
+                    />
+                    <DataTable.Column
+                        label="Deployment Id"
```

Comment:
Can we use 

```suggestion
                        label="Deployment ID"
```

similar to https://github.com/cloudify-cosmo/cloudify-stage/blob/0df9298242f1de6cd7576066302a9f3c6adb7c5b/widgets/executions/src/widget.tsx#L33 ?

Reply:
this was named like other ID fields eg. `Node Id`

Reply:
Hmm... ok, makes sense to have it consistent locally, but then it's inconsistent globally. I don't mind too much